.storybook/bundle.ts

```diff
@@ -0,0 +1,16 @@
+/*
+  This file creates the bundle needed by storybook to import the UI components.
+  All components available in the Storybook must be exported here.
+*/
+
+import Cell from '../src/components/dataset-viewer/Cell.vue'
```

Comment:
This component seems to be part of another PR, you should remove the import here.

Reply:
Oops, it's effectively in the next one ;)